agent/handlers/v2/response_test.go

```diff
@@ -188,3 +188,203 @@ func TestContainerResponse(t *testing.T) {
 		})
 	}
 }
+
+func TestTaskResponseMarshal(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	expectedTaskResponseMap := map[string]interface {}{
+		"Cluster": cluster,
+		"TaskARN": taskARN,
+		"Family": family,
+		"Revision": version,
+		"DesiredStatus": "RUNNING",
+		"KnownStatus": "RUNNING",
+		"Containers": []interface {}{
+			map[string]interface {}{
+				"DockerId": containerID,
+				"Name": containerName,
+				"DockerName": containerName,
+				"Image": imageName,
+				"ImageID": imageID,
+				"Ports": []interface {}{
+					map[string]interface {}{
+						"HostPort": float64(80),
```

Comment:
why floats?

Reply:
Because when we unmarshall something to `map[string]interface{}`, the number will be unmarshalled to `float64` type, make it float here so that we can compare the two maps. This was explained in my [previous PR](https://github.com/aws/amazon-ecs-agent/blob/02a6a9fff9bee5e0bbd2b936d4ceee50207595f8/agent/handlers/v1/response_test.go#L50), it would be redundant if I add comments for every case like this.